engine/src/main/java/org/hibernate/search/engine/search/dsl/projection/SearchProjectionFactoryContext.java

```diff
@@ -72,6 +72,30 @@
 	 */
 	FieldProjectionContext<Object> field(String absoluteFieldPath);
 
+	/**
+	 * Project to a field of the indexed document.
+	 *
+	 * Unlike {@link #field(String, Class)} this method bypass any {@code DslConverter} defined on the field,
+	 * so that the values will be provided exactly as they are stored in the backend.
```

Comment:
Please don't mention DSL converters here. I suppose it's an oversight and you already know this, because I already told you so multiple times on the chat, but DSL converters are unrelated to projection converters:

* DSL converters convert parameters passed to the DSL to index field values, typically from the type of the entity properties, so `JavaUtilDateBridge` would convert a `java.util.Date` to an `Instant`.
* Projection converters convert projected field values to a type expected by the user, typically to the type of the entity properties, so `JavaUtilDateBridge` would convert an `Instant` to a `java.util.date`.

Here is a perhaps better javadoc for this method (please adapt it for the other method below). Ideally we should explain converters more in detail, but that'll have to wait:

```
	/**
	 * Project to a field of the indexed document
	 * <p>
	 * Using this method instead of {@link #field(String, Class)} will disable some of the conversion applied to the field value,
	 * allowing to retrieve values directly from the backend.
	 *
	 * @param absoluteFieldPath The absolute path of the field.
	 * @return A context allowing to define the projection more precisely.
	 */
```

Reply:
Yep quoting `DSL converters` here was just an oversight.

Reply:
I was intending bypass the `DSL converter`'s **dual** => `Projection converter`

Reply:
You didn't update the other `rawField` method... Will do.

Reply:
Thanks!